src/main/java/com/palantir/gradle/gitversion/GitVersionArgs.java

```diff
@@ -3,6 +3,7 @@
 import com.google.common.base.Preconditions;
 
 import java.util.Map;
+import java.util.Objects;
 
 class GitVersionArgs {
```

Comment:
Should we be pulling in Immutables to auto-gen most of this class?

Reply:
yes!